Formula/p/python-free-threading.rb

```diff
@@ -0,0 +1,497 @@
+      ]
+      inreplace bad_cellar_path_files, prefix, opt_prefix
+
+      # Help third-party packages find the Python framework
+      inreplace lib_cellar/"config-#{version.major_minor}t-darwin/Makefile",
+                /^LINKFORSHARED=(.*)PYTHONFRAMEWORKDIR(.*)/,
+                "LINKFORSHARED=\\1PYTHONFRAMEWORKINSTALLDIR\\2"
+
+      # Fix for https://github.com/Homebrew/homebrew-core/issues/21212
+      inreplace lib_cellar/"_sysconfigdata_t_darwin_darwin.py",
+                %r{('LINKFORSHARED': .*?) (PythonT.framework/Versions/3.\d+/PythonT)'}m,
+                "\\1 #{opt_prefix}/Frameworks/\\2'"
+
+    else
+      # Prevent third-party packages from building against fragile Cellar paths
+      inreplace Dir[lib_cellar/"**/_sysconfigdata_t_*linux_x86_64-*.py",
+                    lib_cellar/"config*/Makefile",
+                    bin/"python#{version.major_minor}t-config",
+                    lib/"pkgconfig/python-3*.pc"],
+                prefix, opt_prefix
+
+      inreplace bin/"python#{version.major_minor}t-config",
+                'prefix_real=$(installed_prefix "$0")',
+                "prefix_real=#{opt_prefix}"
+    end
+
+    # Remove the site-packages that Python created in its Cellar.
+    rm_r(site_packages_cellar)
+
+    # Prepare a wheel of wheel to install later.
+    common_pip_args = %w[
+      -v
+      --no-deps
+      --no-binary :all:
+      --no-index
+      --no-build-isolation
+    ]
+    whl_build = buildpath/"whl_build"
+    system python3, "-m", "venv", whl_build
+    %w[flit-core wheel setuptools].each do |r|
+      resource(r).stage do
+        system whl_build/"bin/pip3", "install", *common_pip_args, "."
+      end
+    end
+    resource("wheel").stage do
+      system whl_build/"bin/pip3", "wheel", *common_pip_args,
+                                            "--wheel-dir=#{libexec}",
+                                            "."
+    end
+
+    # Replace bundled pip with our own.
+    rm lib_cellar.glob("ensurepip/_bundled/pip-*.whl")
+    %w[pip].each do |r|
+      resource(r).stage do
+        system whl_build/"bin/pip3", "wheel", *common_pip_args,
+                                              "--wheel-dir=#{lib_cellar}/ensurepip/_bundled",
+                                              "."
+      end
+    end
+
+    # Patch ensurepip to bootstrap our updated version of pip
+    inreplace lib_cellar/"ensurepip/__init__.py" do |s|
+      s.gsub!(/_PIP_VERSION = .*/, "_PIP_VERSION = \"#{resource("pip").version}\"")
+    end
+
+    # Write out sitecustomize.py
+    (lib_cellar/"sitecustomize.py").atomic_write(sitecustomize)
+
+    # Rename idle, pydoc to t variants
+    mv bin/"idle#{version.major_minor}", bin/"idle#{version.major_minor}t"
+    mv bin/"pydoc#{version.major_minor}", bin/"pydoc#{version.major_minor}t"
+
+    # Remove python3.13 named executables
+    rm_r bin/"python3"
+    rm_r bin/"python3.13"
+    rm_r bin/"python3.13-config"
```

Comment:
You still have `idle3`, `pydoc3` and `python3-config` here. Let's just remove everything that doesn't have `3.13t` in it:
```suggestion
    (bin.children - bin.glob("*#{version.major_minor}t*")).map(&:unlink)
```